test/branded_token/constructor.js

```diff
@@ -20,116 +20,116 @@ const utils = require('../test_lib/utils');
 const BrandedToken = artifacts.require('BrandedToken');
 
 contract('BrandedToken::constructor', async () => {
```

Comment:
"use strict" should be added after LICENSE. It's a global comment to add in files wherever missing.

Reply:
Done.